test/src/unit/mwc-textfield.test.ts

```diff
@@ -203,4 +203,27 @@ suite('mwc-textfield:', () => {
       }
     });
   });
+
+  suite('select', () => {
+    let element: TextField;
+
+    setup(async () => {
+      fixt = await fixture(basic);
+
+      element = fixt.root.querySelector('mwc-textfield')!;
+    });
+
+    test('selects the input text', () => {
+      const input = element.shadowRoot!.querySelector('input');
+
+      element.select();
+      assert.equal(element.shadowRoot!.activeElement, input);
```

Comment:
MDN says that not all browsers focus the selection. Perhaps use `getSelection`?

Reply:
seems there's some funkiness with `getSelection` on the window/root. it doesn't work with inputs well if at all (googled a bit and seemed to say you should just access the input's start/end ranges directly).

Reply:
i could maybe just `focus` before the select, here in the test?

Reply:
That wouldn't necessarily test select but focus. I think your test for setSelectionRange would also work here